common/predictive-text/polyfills/array.from.js

```diff
@@ -0,0 +1,81 @@
+// From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from
+// Any npm-based ones require use of `require`, which won't work for us.
+
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
+if (!Array.from) {
```

Comment:
Fixing the first issue wasn't enough for a full fix in IE; this polyfill was also necessary.  Note the following two links:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/codePointAt#Browser_compatibility

`Array.from` requires a more modern Chrome for Android than `String.charCodeAt`... so it's almost certain this is also necessary for Android support.  (It also gets IE fully off the ground for the unit tests, so yay!)